src/actors/_actorsheets.scss

```diff
@@ -264,4 +264,46 @@
       text-align: center;
     }
   }
+
+  input.characteristic-rank-high {
+    background-color: #018a31FF; //100%
+    color: palegoldenrod;
+  }
+  input.characteristic-rank-10 {
+    background-color: #018a31D6; //84%
+  }
+  input.characteristic-rank-9 {
+    background-color: #018a31AD; //68%
+  }
+  input.characteristic-rank-8 {
+    background-color: #018a3182; //51%
+  }
+  input.characteristic-rank-7 {
+    background-color: #018a3159; //35%
+  }
+  input.characteristic-rank-6 {
+    background-color: #018A3130; //19%
+  }
+  input.characteristic-rank-5 {
+    background-color: #018A310D; //5%
+  }
+  input.characteristic-rank-4 {
+    background-color: #8A01010D; //5%
+  }
+  input.characteristic-rank-3 {
+    background-color: #8A010130; //19%
+  }
+  input.characteristic-rank-2 {
+    background-color: #8A010659; //35%
+  }
+  input.characteristic-rank-1 {
+    background-color: #8A010182; //51%
+  }
+  input.characteristic-rank-0 {
+    background-color: #8A0101AD; //68%
+  }
+  input.characteristic-rank-low {
+    background-color: #8A0101FF;
+    color: palegoldenrod;
+  }
```

Comment:
What do you think about something like this? Same colors, just not as a background.
```suggestion
  input.characteristic-rank-high {
    border-bottom: 7px solid #018a31FF; //100%
    border-left: 5px solid transparent;
    border-right: 5px solid transparent;
  }
  input.characteristic-rank-10 {
    border-bottom: 7px solid #018a31D6; //84%
    border-left: 5px solid transparent;
    border-right: 5px solid transparent;
  }
  input.characteristic-rank-9 {
    border-bottom: 7px solid #018a31AD; //68%
    border-left: 5px solid transparent;
    border-right: 5px solid transparent;
  }
  input.characteristic-rank-8 {
    border-bottom: 7px solid #018a3182; //51%
    border-left: 5px solid transparent;
    border-right: 5px solid transparent;
  }
  input.characteristic-rank-7 {
    border-bottom: 7px solid #018a3159; //35%
    border-left: 5px solid transparent;
    border-right: 5px solid transparent;
  }
  input.characteristic-rank-6 {
    border-bottom: 7px solid #018A3130; //19%
    border-left: 5px solid transparent;
    border-right: 5px solid transparent;
  }
  input.characteristic-rank-5 {
    border-bottom: 7px solid #018A310D; //5%
    border-left: 5px solid transparent;
    border-right: 5px solid transparent;
  }
  input.characteristic-rank-4 {
    border-bottom: 7px solid #8A01010D; //5%
    border-left: 5px solid transparent;
    border-right: 5px solid transparent;
  }
  input.characteristic-rank-3 {
    border-bottom: 7px solid #8A010130; //19%
    border-left: 5px solid transparent;
    border-right: 5px solid transparent;
  }
  input.characteristic-rank-2 {
    border-bottom: 7px solid #8A010659; //35%
    border-left: 5px solid transparent;
    border-right: 5px solid transparent;
  }
  input.characteristic-rank-1 {
    border-bottom: 7px solid #8A010182; //51%
    border-left: 5px solid transparent;
    border-right: 5px solid transparent;
  }
  input.characteristic-rank-0 {
    border-bottom: 7px solid #8A0101AD; //68%
    border-left: 5px solid transparent;
    border-right: 5px solid transparent;
  }
  input.characteristic-rank-low {
    border-bottom: 7px solid #8A0101FF;
    border-left: 5px solid transparent;
    border-right: 5px solid transparent;
  }
```